modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -4,19 +4,23 @@
 require 'token_validation/v2/client'
 require 'claims_api/claim_logger'
 require 'claims_api/dependent_service'
+require 'claims_api/v2/params_validation/evidence_waiver_submission'
 
 module ClaimsApi
   module V2
     module Veterans
       class EvidenceWaiverController < ClaimsApi::V2::Veterans::Base
         skip_before_action :validate_json_format
         before_action :set_lighthouse_claim, :set_bgs_claim!, :verify_if_dependent_claim!
+        FORM_NUMBER = '5103'
 
         def submit
+          validate_request!(ClaimsApi::V2::ParamsValidation::EvidenceWaiverSubmission)
           validate_veteran_name(false)
 
+          tracked_item_ids = params['data']['attributes']['trackedItems'] if params['data'].present?
           ews = create_ews(params[:id])
-          ClaimsApi::EvidenceWaiverBuilderJob.perform_async(ews.id, @pctpnt_vet_id)
+          ClaimsApi::EvidenceWaiverBuilderJob.perform_async(ews.id, tracked_item_ids, @pctpnt_vet_id)
```

Comment:
thoughts on adding tracked items to the EWS in the DB? think I'm leaning towards having platform run a migration to add these to the table and using that, then we wouldn't need to pass it, we could just simply provide the ews id and extract the ID's in the job. 

Feedback, thoughts, other ideas?

Reply:
@FonzMP Yes, I like that idea